src/balanced_bagging.jl

```diff
@@ -27,7 +27,7 @@ function get_majority_minority_inds_counts(y)
     # a tuple mapping each class to its indices
     labels_inds = collect(group_inds(y))
     num_classes = length(labels_inds)
-    num_classes == 2 || throw(ArgumentError(ERR_MULTICLASS_UNSUPP(num_classes)))
+    num_classes == 2 || throw(ErrorException(ERR_MULTICLASS_UNSUPP(num_classes)))
```

Comment:
@EssamWisam 

The constant is already an exception, so we don't need to wrap it. So something like this?

```suggestion
    num_classes == 2 || throw(ERR_MULTICLASS_UNSUPP(num_classes))
```

Reply:
@ablaom I usually prefer ArgumentError because it prints `ArgumentError: message` instead of `message`; otherwise, it may take time for the user to realize that it is an error. Remedy is as easy as inserting "Error: " in each message; especially since ArgumentError tests behave differently in Julia 1.7